web/packages/teleterm/src/main.ts

```diff
@@ -47,14 +47,23 @@ function initializeApp(): void {
     filePath: path.join(settings.userDataDir, 'app_state.json'),
     debounceWrites: true,
   });
-  const appConfigFileStorage = createFileStorage({
+  const configFileStorage = createFileStorage({
     filePath: path.join(settings.userDataDir, 'app_config.json'),
     debounceWrites: false,
   });
-  const configService = createConfigService(
-    appConfigFileStorage,
-    settings.platform
-  );
+  const configJsonSchemaFileStorage = createFileStorage({
```

Comment:
`configJsonSchemaFileStorage` is used solely in `configService`.

`configService` already knows what the path of the config file is (thanks to the new function that you added).

What do you think about moving the creation of this file to `createConfigService` instead?

Reply:
But how `configJsonSchemaFileStorage` would know about the user data dir? If I understand your suggestion correctly, you would like to take a dir name of `configFileStorage` in `configService` and create a file based on it?

Reply:
> But how `configJsonSchemaFileStorage` would know about the user data dir? If I understand your suggestion correctly, you would like to take a dir name of `configFileStorage` in `configService` and create a file based on it?

Yes, `configJsonSchemaFileStorage` could assume that the json schema lives next to the file that is described by the schema.

Reply:
We could do it, but if we move `jsonSchemaFile` to `createConfigStore` we won't be able to use it in the tests (I mean spying on the `replace` function).